{"name": "test_create_with_invalid_data[book_data0]", "status": "passed", "description": "Проверяем, что нельзя создать книгу с некорректными данными, нарушиющих типы", "parameters": [{"name": "book_data", "value": "(2, 'asd')"}], "start": 1676239552881, "stop": 1676239552890, "uuid": "ef5a53c7-5d6d-4e16-9236-0567badf7b7e", "historyId": "b6b684fca315a9b10e94241db5295068", "testCaseId": "0d2a4e16b7c113dbdc5907d1b75a2e47", "fullName": "test_API#test_create_with_invalid_data", "labels": [{"name": "tag", "value": "negative"}, {"name": "suite", "value": "test_API"}, {"name": "host", "value": "jenkins"}, {"name": "thread", "value": "811-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_API"}]}